File: app/src/main/java/com/rdev/tryp/model/NearbyDriver.java
package com.rdev.tryp.model;

import com.google.gson.annotations.SerializedName;

public class NearbyDriver{

	@SerializedName("data")
	private Data data;

	public void setData(Data data){
		this.data = data;
	}

	public Data getData(){
		return data;
	}

	@Override
 	public String toString(){
		return 
			"NearbyDriver{" + 
			"data = '" + data + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/rdev/tryp/model/NearbyDriver.kt
package com.rdev.tryp.model

import com.google.gson.annotations.SerializedName


class NearbyDriver {

    @SerializedName("data")
    var data: Data? = null

    override fun toString(): String {
        return "NearbyDriver{" +
                "data = '" + data + '\''.toString() +
                "}"
    }

}